lib/publiccloud/utils.pm

```diff
@@ -146,6 +146,10 @@ sub register_openstack {
 # Validation for update repos
 sub validate_repo {
     my ($maintrepo) = @_;
+    if (is_sle_micro('>=6.0')) {
+        record_info("NEW MAINT MODE", "New style update, so can't validate repository");
```

Comment:
Do we really want one info box per repository?

Reply:
on SL Micro 6 its only one repo anyway